passivbot.py

```diff
@@ -86,11 +86,13 @@ def set_config(self, config):
             config['last_price_diff_limit'] = 0.3
         if 'profit_trans_pct' not in config:
             config['profit_trans_pct'] = 0.0
+        if 'assigned_balance' not in config:
+            config['assigned_balance'] = None
         if 'cross_wallet_pct' not in config:
             config['cross_wallet_pct'] = 1.0
         if config['cross_wallet_pct'] > 1.0 or config['cross_wallet_pct'] <= 0.0:
             print(f'An invalid value is provided for `cross_wallet_pct` ({config["cross_wallet_pct"]}). The value must be bigger than 0.0 and less than or equal to 1.0. The'
-                  f'bot will start with the default value of 1.0, meaning it will utilize the ')
+                  f'bot will start with the default value of 1.0, meaning it will utilize the full wallet balance available.')
```

Comment:
@enarjord with the changes you made, I don't think this applies anymore, since cross_wallet_pct is now also applied to assigned_balance?

Reply:
it applies when assigned_balance is None

Reply:
that's not part of the if-statement though, so if assigned_balance is set to say $500 (in a wallet of $1000), it won't actually be using the 'full wallet balance available' as stated in the print. Although, one could argue that 'full wallet balance available' is manipulated by assigned_balance. Happy to leave it as is, just something I noticed.